### DISCUSSION TITLE
hold_keys(q) repeats when used with mouse button

### DISCUSSION THREAD

**Discussion by heartburn22** [sezanzeb/input-remapper#461] [General]
I'm configuring a keypad and mouse to use with a game. I got everything working great except for one thing: If I map  WheelLeft on my mouse to hold_keys(q) (or anything for that matter), it pulses the key instead of holding it. If I do the same mapping to a key on my keypad, it works fine -- no pulsing. It only seems to happen with the mouse buttons.

Thanks for any help!

**Comment by jonasBoss** [comment]
This is most likely because the Scroll wheel is not a button (yes even if it is on your mouse) in order to scroll the page sideways the mouse repeats the scroll event many times a second.
Inputremapper waits a bit (`Release Timeout`) for the scroll wheel to repeat its signal in order to avoid the issue you described above, but if your mouse is slow to repeat the scroll signal it will pulse the mapping.

The [beta](https://github.com/sezanzeb/input-remapper/tree/beta) branch has a setting for the `Release Timeout`. If you install the beta version you can set the `Release Timeout` to a higher value until the mapping no longer pulses.
![image](https://user-images.githubusercontent.com/31804124/186611540-4c97a48d-604b-4b69-a355-7101e8460005.png)

**Reply by heartburn22** [reply]
Thanks! I give it a try.
